specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01/actionGroups_API.json

```diff
@@ -0,0 +1,862 @@
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The request succeeded.",
+                        "schema": {
+                            "$ref": "#/definitions/ActionGroupList"
+                        }
+                    },
+                    "default": {
+                        "description": "An error occurred and the list of action groups could not be retrieved.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/microsoft.insights/actionGroups/{actionGroupName}/subscribe": {
+            "post": {
+                "description": "Enable a receiver in an action group. This changes the receiver's status from Disabled to Enabled. This operation is only supported for Email or SMS receivers.",
+                "operationId": "ActionGroups_EnableReceiver",
+                "x-ms-examples": {
+                    "Enable the receiver": {
+                        "$ref": "./examples/enableReceiver.json"
+                    }
+                },
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/ResourceGroupNameParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ActionGroupNameParameter"
+                    },
+                    {
+                        "name": "enableRequest",
+                        "in": "body",
+                        "required": true,
+                        "schema": {
+                            "$ref": "#/definitions/EnableRequest"
+                        },
+                        "description": "The receiver to re-enable."
+                    },
+                    {
+                        "$ref": "#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The receiver was successfully enabled."
+                    },
+                    "409": {
+                        "description": "The receiver is already enabled in the action group."
+                    },
+                    "default": {
+                        "description": "An error occurred and the receiver could not be enabled, e.g.: 404: The action group was not found or no matching receiver was found in the action group.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Resource": {
```

Comment:
FYI - there are common types defined here: https://github.com/Azure/azure-rest-api-specs/tree/master/specification/common-types/resource-management/v1  you can use those rather than duplicating.